Doc/whatsnew/3.10.rst

```diff
@@ -439,6 +439,11 @@ New Features
 * The :c:func:`PyType_GetSlot` function can accept static types.
   (Contributed by Hai Shi and Petr Viktorin in :issue:`41073`.)
 
+* The ``Py_TRASHCAN_BEGIN`` macro no longer accesses PyTypeObject attributes,
+  but now can get the condition by calling the new private
+  :c:func:`_PyTrash_cond()` function which hides implementation details.
+  (Contributed by Hai Shi in :issue:`40170`.)
```

Comment:
I don't think that it's worth it to mention in What's New in Python 3.10. End users will not notice and don't care ;-)

Reply:
Ok, removed.
I decided add this news because I saw you have written the updated info of `_PyTrash_begin()` and `_PyTrash_end()`, Lol~

Reply:
I wrote a changelog entry, but nothing in What's New in Python 3.9 or What's New in Python 3.10, no?

Reply:
Oh, I saw this info in https://github.com/python/cpython/blob/master/Misc/NEWS.d/3.9.0a5.rst.
Looks like I confused the news file with relese file : (, sorry.

Reply:
No problem.

I consider that "What's New in Python X.Y" should be short and only describe the public API, but the Changelog should describe every single change and private functions can be mentioned there.

Reply:
Oh, This great. I am very confused when I start  write the whatsnew file in the firsttime.
It's more like a double copy operation. 

In fact, I don't know release manager how to archieve those change log.
My simple thought: Maybe we can create some labels to tag what's info should be added  in release file or not. When release manager archive those change log, those taged changeinfo chould be archieved in release file automatically.

If this thought have any value, I can add this in my TODO list. Lol~